compiler-rt/test/profile/instrprof-binary-correlate-debuginfod.c

```diff
@@ -0,0 +1,40 @@
+// REQUIRES: (linux || windows), curl
+// RUN: rm -rf %t
+
+// Default instrumented build with no profile correlation.
+// RUN: %clang_profgen -o %t.default.exe -Wl,--build-id=0x12345678 -fprofile-instr-generate -fcoverage-mapping %S/Inputs/instrprof-debug-info-correlate-main.cpp %S/Inputs/instrprof-debug-info-correlate-foo.cpp
+// RUN: env LLVM_PROFILE_FILE=%t.default.profraw %run %t.default.exe
+// RUN: llvm-profdata merge -o %t.default.profdata %t.default.profraw
+// RUN: llvm-profdata show %t.default.profdata > %t.default.profdata.show
```

Comment:
```suggestion
// RUN: llvm-profdata show --all-functions --counts %t.default.profdata > %t.default.profdata.show
```
Make it consistent with the debug-info test. Plain `llvm-profdata show` doesn't show much information.

Reply:
Done.